src/main/java/foxtrot/uniform/charlie/kilo/model/MovieSession.java

```diff
@@ -0,0 +1,70 @@
+package foxtrot.uniform.charlie.kilo.model;
+
+import java.time.LocalDateTime;
+import javax.persistence.Entity;
+import javax.persistence.FetchType;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.ManyToOne;
+import javax.persistence.Table;
+
+@Entity
+@Table(name = "movieSession")
+public class MovieSession {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    @ManyToOne (fetch = FetchType.LAZY)
+    @JoinColumn(name = "movie_id")
+    private Movie movie;
+    @ManyToOne (fetch = FetchType.LAZY)
+    @JoinColumn(name = "cinemaHall_id")
+    private CinemaHall cinemaHall;
+    private LocalDateTime showTime;
```

Comment:
```suggestion
    private LocalDateTime showTime;
```
use here JoinColumn() also

Reply:
I agree with Max here